tests/e2e/tools/lib/post.js

```diff
@@ -1,3 +1,5 @@
+import {expect} from '@playwright/test';
```

Comment:
We should use the `expect` coming from our [test utils](https://github.com/greenpeace/planet4-master-theme/blob/main/tests/e2e/tools/lib/test-utils.js#L29)

Reply:
After [this requested change](https://github.com/greenpeace/planet4-master-theme/pull/2419#discussion_r1873420675), we don't need to import the `expect` function through this file.